### ISSUE TITLE
Publish lsp4s for scala 2.11

### ISSUE THREAD

**Issue by ShaneDelmore** [scalameta/metals#189]
I'm using and enjoying lsp4s, but as I am currently stuck on 2.11 I have copied and pasted it into my local project and would prefer to just use a published version compatible with 2.11

**Comment by gabro** [comment]
> I don't think we use any 2.12-only features in lsp4s

Almost 😛 Remember when I pushed you to use a lot of SAMs? 

See #190

**Comment by olafurpg** [comment]
Fixed in #190 , CI should release to Bintray once tests complete after merge @ShaneDelmore

**Comment by ShaneDelmore** [comment]
I'm having no luck with this.  Am I doing something wrong? 
In sbt I have added 

resolvers += Resolver.bintrayRepo("scalameta", "maven")

but still neither of these work:

libraryDependencies ++= List(
  "org.scalameta" %% "lsp4s" % "325610ac",
  "org.scalameta" %% "jsonrpc" % "325610ac"
)

**Comment by olafurpg** [comment]
I don't think the CI is configured to release for 2.11, 2.12 downloads OK

```
 coursier fetch org.scalameta:lsp4s_2.12:68015934 -r bintray:scalameta/maven
```

https://bintray.com/scalameta/maven/lsp4s/68015934

Can you check the .travis.yml?

**Comment by gabro** [comment]
It's probably a missing + in front of releaseEarly

--
Gabriele

**Comment by olafurpg** [comment]
`+` probably works actually because we are using sbt 1.0 where sbt-doge is no longer needed 🙏 

let's check if it works now https://travis-ci.org/scalameta/metals/builds/339421516

**Comment by olafurpg** [comment]
lsp4s is now released for 2.11
```
coursier fetch org.scalameta:lsp4s_2.11:6f55b052 -r bintray:scalameta/maven
```

Note that we have not tested it for 2.11, only 2.12. See #191

**Comment by laughedelic** [comment]
@olafurpg sorry, I couldn't reply before. But yes, you are right, sbt-doge got "merged" into sbt 1.0, so `+` should work for relevant subprojects without extra effort.
